src/udev/scsi_id/scsi_serial.c

```diff
@@ -765,11 +976,58 @@ int scsi_std_inquiry(struct scsi_id_device *dev_scsi, const char *devname) {
         format_devnum(statbuf.st_rdev, dev_scsi->kernel);
 
         memzero(buf, SCSI_INQ_BUFF_LEN);
-        err = scsi_inquiry(dev_scsi, fd, 0, 0, buf, SCSI_INQ_BUFF_LEN);
+        err = sgio_scsi_inquiry(dev_scsi, fd, 0, 0, buf, SCSI_INQ_BUFF_LEN);
+        if (err < 0)
+                goto out;
```

Comment:
Then, this is not necessary. simply return `err`.

Reply:
Done